### PR TITLE
Use `execFileSync` instead of `sync-exec`

### LINKED ISSUE #3: Use `execFileSync`
Use the native `execFileSync` when Node 0.12 is the minimum required version.

https://github.com/sindresorhus/passwd-user/blob/master/index.js#L4

### PR DESCRIPTION
**Pull request by kevva**
Probably needs to be tested on OS X first.

Fixes #3.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -103,7 +103,10 @@ module.exports.sync = function (username) {
 	if (process.platform === 'linux') {
 		return getUser(fs.readFileSync('/etc/passwd', 'utf8'), username);
 	} else if (process.platform === 'darwin') {
-		return getUser(execSync('/usr/bin/id -P "' + username + '"').stdout, username);
+		return getUser(execFileSync('/usr/bin/id', ['-P', '"' + username + '"'], {
+			encoding: 'utf8',
+			stdio: ['ignore', 'pipe', 'ignore']
```
Comment (kevva) [inline index.js]: Ignores `stdin` and `stderr`.

**Comment by sindresorhus** [comment]
Too early to require Node 0.12 unfortunately. Can you conditionalize it?

**Comment by kevva** [comment]
You could release it as a major, but might be better to conditionalize it for now.

**Comment by sindresorhus** [comment]
Feature test if `execFileSync` is available.

**Comment by kevva** [comment]
Ah, of course.

File: index.js [hunk 2]
```diff
@@ -103,6 +104,13 @@ module.exports.sync = function (username) {
 	if (process.platform === 'linux') {
 		return getUser(fs.readFileSync('/etc/passwd', 'utf8'), username);
 	} else if (process.platform === 'darwin') {
+		if (typeof execFileSync === 'function') {
+			return getUser(execFileSync('/usr/bin/id', ['-P', '"' + username + '"'], {
```
Comment (sindresorhus) [inline index.js]: You don't need the `"` when using `execFileSync`.

Comment (kevva) [inline index.js]: Done.

**Pull request merged**